sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -179,3 +180,46 @@ private[sql] trait SQLTestUtils
     DataFrame(_sqlContext, plan)
   }
 }
+
+private[sql] object SQLTestUtils {
```

Comment:
Why put this in an object instead of in the already existing trait?  It just makes invocation more verbose.